src/main/java/org/alliancegenome/curation_api/controllers/crud/AGMDiseaseAnnotationCrudController.java

```diff
@@ -50,37 +50,37 @@ public APIResponse updateAgmDiseaseAnnotations(String taxonID, List<AGMDiseaseAn
 
 	@Override
 	public APIResponse updateZfinAgmDiseaseAnnotations(List<AGMDiseaseAnnotationDTO> annotations) {
-		return agmDiseaseAnnotationExecutor.runLoad("NCBITaxon:7955", annotations);
+		return agmDiseaseAnnotationExecutor.runLoad("Danio rerio", annotations);
 	}
 
 	@Override
 	public APIResponse updateMgiAgmDiseaseAnnotations(List<AGMDiseaseAnnotationDTO> annotations) {
-		return agmDiseaseAnnotationExecutor.runLoad("NCBITaxon:10090", annotations);
+		return agmDiseaseAnnotationExecutor.runLoad("Mus musculus", annotations);
 	}
 
 	@Override
 	public APIResponse updateRgdAgmDiseaseAnnotations(List<AGMDiseaseAnnotationDTO> annotations) {
-		return agmDiseaseAnnotationExecutor.runLoad("NCBITaxon:10116", annotations);
+		return agmDiseaseAnnotationExecutor.runLoad("Rattus norvegicus", annotations);
 	}
 
 	@Override
 	public APIResponse updateFbAgmDiseaseAnnotations(List<AGMDiseaseAnnotationDTO> annotations) {
-		return agmDiseaseAnnotationExecutor.runLoad("NCBITaxon:7227", annotations);
+		return agmDiseaseAnnotationExecutor.runLoad("Drosophila melanogaster", annotations);
 	}
 
 	@Override
 	public APIResponse updateWbAgmDiseaseAnnotations(List<AGMDiseaseAnnotationDTO> annotations) {
-		return agmDiseaseAnnotationExecutor.runLoad("NCBITaxon:6239", annotations);
+		return agmDiseaseAnnotationExecutor.runLoad("Caenorhabditis elegans", annotations);
 	}
 
 	@Override
 	public APIResponse updateHumanAgmDiseaseAnnotations(List<AGMDiseaseAnnotationDTO> annotations) {
-		return agmDiseaseAnnotationExecutor.runLoad("NCBITaxon:9606", annotations);
+		return agmDiseaseAnnotationExecutor.runLoad("Homo sapiens", annotations);
 	}
 
 	@Override
 	public APIResponse updateSgdAgmDiseaseAnnotations(List<AGMDiseaseAnnotationDTO> annotations) {
-		return agmDiseaseAnnotationExecutor.runLoad("NCBITaxon:559292", annotations);
+		return agmDiseaseAnnotationExecutor.runLoad("Saccharomyces cerevisiae", annotations);
```

Comment:
@cmpich @markquintontulloch are these endpoints even used? Sense we now load all these via bulk loads?

Reply:
Used in integration tests